### PR TITLE
Force target to be java 8 even if we are using jdk 11 to compile

### PR DESCRIPTION
**Pull request by yuzhu**
We need our final artifact to be compatible with java 8 even if we are running jdk 11 compiler.

### CODE REVIEW

**Comment by yuzhu** [comment]
jenkins, test this with java 11 please

File: pom.xml [hunk 1]
```diff
@@ -1304,7 +1304,7 @@
         <jdk>11</jdk>
       </activation>
       <properties>
-        <java.version>11</java.version>
+        <java.version>8</java.version>
```
Comment (gpang) [inline pom.xml:1307]: Does this set the target to 8? Is it supposed to 8, or 1.8, or does it not matter?

Comment (yuzhu) [reply pom.xml]: oops, fixed

**Comment by yuzhu** [comment]
@gpang PTAL

**Comment by gpang** [comment]
@yuzhu I think this is good. @ZacBlanco can you verify, since you initially added this block?

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Pull request merged**